server/status/feed.go

```diff
@@ -68,16 +68,20 @@ func (nef NodeEventFeed) StartNode(desc proto.NodeDescriptor, startedAt int64) {
 // CallComplete is called by a node whenever it completes a request. This will
 // publish an appropriate event to the feed based on the results of the call.
 func (nef NodeEventFeed) CallComplete(args proto.Request, reply proto.Response) {
```

Comment:
yes, as we move to batch requests everywhere I want to phase out `proto.{Request,Response}` where possible.